stream_alert_cli/apps/handler.py

```diff
@@ -13,97 +13,232 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+import string
+
 from stream_alert.apps import StreamAlertApp
 from stream_alert.shared.logger import get_logger
 from stream_alert_cli.apps.helpers import save_app_auth_info
+from stream_alert_cli.utils import (
+    CliCommand, set_parser_epilog, generate_subparser,
+    add_schedule_expression_arg, add_timeout_arg, add_memory_arg,
+    CLUSTERS)
```

Comment:
hanging `)` should be on new line

Reply:
it's also good practice to keep imports in alphabetical order

Reply:
Sorry, I miss these a lot because my IDE auto-generates these imports and also auto-hides them

Reply:
bad IDE. BAD!